### PR TITLE
Bug 1373032, added a new Redeploying Certificates topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1373032

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@abutcher PTAL. 
@adellape @bfallonf Please peer review. Thanks!

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+This topic reviews how to back up and redeploy cluster certificates using
+`ansible-playbook`. This method fixes common certificate errors. Possible use
```
Comment (bfallonf) [inline install_config/redeploying_certificates.adoc]: Would it be more accurate to say something like "ansible-playbook command" or "Ansible playbook"?

File: install_config/redeploying_certificates.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+cases include:
+
+- The installer detected the wrong host names and the issue was identified too late.
+- The certificates are expired and you just want to update them.
```
Comment (bfallonf) [inline install_config/redeploying_certificates.adoc]: I'd suggest s/just want to/need to

**Comment by bfallonf** [comment]
@ahardin-rh A couple suggestions, but god otherwise :tada:

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../ install_config/redeploying_certificates.adoc#install-config-redeploying-certificates [Redeploying Certificates]
|New topic reviewing how to back up and redeploy cluster certificates using the `ansible-playbook` command.
%

**Pull request merged**